rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly"
```

Comment:
hm, not sure we want to default to the nightly channel

Reply:
I just noticed, as I tried setting this back to stable, it caused the issue I mentioned I had fixed by reverting to the old code mentioend in the comment here:
https://github.com/LNP-BP/descriptor-wallet/pull/27#discussion_r812304388

I'm not sure we can get this to pass clippy lints on stable at this time without also making breaking changes to dependents.

Reply:
It seems that we can again move to stable with the latest 0.59 release

Reply:
It's true, how convenient :)